### PR TITLE
GSIP-225 update mailing list references

### PR DESCRIPTION
**Pull request by petersmythe**
[GSIP-225](https://osgeo-org.atlassian.net/browse/GSIP-225) [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=geoserver&utm_campaign=badge_info)

DO NOT MERGE YET!

* [ ] https://github.com/geoserver/geoserver.github.io/pull/208 website update
* [x] https://discourse.osgeo.org/c/geoserver/geoserver-users/51 available
* [x] welcome post https://discourse.osgeo.org/t/welcome-to-geoserver-user-forum/31009 
* [ ] https://trac.osgeo.org/osgeo/ticket/3104 (geoserver-users group needs to be renamed for consistency)
* [x] establish email address, presently geoserver-user@discourse.osgeo.org but we expected geoserver-user@osgeo.org 
* [x] fix spelling of `channels` in https://discourse.osgeo.org/t/welcome-to-geoserver-user-forum/31009

# Checklist

- [ ] I have read the [contribution guidelines](https://github.com/geoserver/geoserver/blob/main/CONTRIBUTING.md).
- [ ] I have sent a [Contribution Licence Agreement](https://docs.geoserver.org/latest/en/developer/policies/committing.html) (not required for small changes, e.g., fixing typos in documentation).
- [ ] First PR targets the `main` branch (backports managed later; ignore for branch specific issues).
- [ ] All the build checks are green ([see automated QA checks](https://docs.geoserver.org/latest/en/developer/qa-guide/index.html)).

For core and extension modules:

- [ ] New unit tests have been added covering the changes.
- [ ] [Documentation](https://github.com/geoserver/geoserver/tree/main/doc/en/user/source) has been updated (if change is visible to end users).
- [ ] The [REST API docs](https://github.com/geoserver/geoserver/tree/main/doc/en/api/1.0.0) have been updated (when changing configuration objects or the REST controllers).
- [ ] There is an issue in the [GeoServer Jira](https://osgeo-org.atlassian.net/browse/GEOS/summary) (except for changes that do not affect administrators or end users in any way).
- [ ] Commit message(s) must be in the form ``[GEOS-XYZWV] Title of the Jira ticket``.
- [ ] Bug fixes and small new features are presented as a single commit.
- [ ] Each commit has a single objective (if there are multiple commits, each has a separate JIRA ticket describing its goal).

### CODE REVIEW

**Comment by petersmythe** [comment]
* [ ] Outstanding:  https://geoserver.org/comm/userlist-guidelines.html

**Comment by jodygarnett** [comment]
Rebasing to pass QA tests. The discourse channel is now live so we can complete this PR: https://discourse.osgeo.org/c/geoserver/geoserver-users/51

File: doc/en/api/1.0.0/gwcglobal.yaml [hunk 1]
```diff
@@ -6,8 +6,8 @@ info:
   description: Global configuration for the GeoWebCache server
   contact:
     name: GeoServer
-    email: 'geoserver-users@sourceforge.net'
-    url: 'http://geoserver.org/comm/'
+    email: 'geoserver-users@osgeo.org'
```
Comment (jodygarnett) [inline doc/en/api/1.0.0/gwcglobal.yaml:9]: Oh this is interesting, send email will not be accepted unless they have subscribed I don't think?

**Comment by petersmythe** [comment]
> Rebasing to pass QA tests. The discourse channel is now live so we can complete this PR: https://discourse.osgeo.org/c/geoserver/geoserver-users/51
> 
> It looks a little odd with just one sub category :P

I am glad that https://discourse.osgeo.org/c/geoserver/geoserver-users redirects, so let's stick with this form (without the /51 at the end)

File: doc/en/api/1.0.0/gwcglobal.yaml (hunk 1 above)
Comment (petersmythe) [reply doc/en/api/1.0.0/gwcglobal.yaml:9]: Or should the email address be geoserver-user@discourse.osgeo.org?

image

**Comment by petersmythe** [comment]
Hmm, why can I not reply to https://discourse.osgeo.org/t/testing-new-topic-by-mail/49596 - there is no reply button.  Yet @jodygarnett managed to reply to Vicky.

**Comment by jodygarnett** [comment]
With discourse live this PR can now be reviewed

**Comment by jodygarnett** [comment]
> Hmm, why can I not reply to https://discourse.osgeo.org/t/testing-new-topic-by-mail/49596 - there is no reply button. Yet @jodygarnett managed to reply to Vicky.

You did not join the group yet? https://discourse.osgeo.org/g/geoserver-users please join and test the instructions

File: doc/en/api/1.0.0/gwcglobal.yaml (hunk 1 above)
Comment (jodygarnett) [reply doc/en/api/1.0.0/gwcglobal.yaml:9]: I just tested by sending an email to [geoserver-user@discourse.osgeo.org](mailto:geoserver-user@discourse.osgeo.org) which is not what we expected.

In earlier testing they setup a simple address (for the marketing committee). I would like to ask them to do the same here.

But yes we need to establish what this is intended to be :)

**Comment by petersmythe** [comment]
> Rebasing to pass QA tests. The discourse channel is now live so we can complete this PR: https://discourse.osgeo.org/c/geoserver/geoserver-users/51
> 
> It looks a little odd with just one sub category :P

My suggestion is to create the `developer` sub category as well, as a placeholder, and make it closed.  Hmm, a group can be closed, not sure about a sub category.  And have the description redirect to the current SourceForge mailing list landing page.

**Pull request merged**